src/dmd/frontend.d

```diff
@@ -140,9 +140,10 @@ void initDMD(
     }
 
     versionIdentifiers.each!(VersionCondition.addGlobalIdent);
-    addDefaultVersionIdentifiers(global.params, target);
```

Comment:
No idea why this was called twice, once before initializing `target`, and then once again after initializing it. AFAICT, `target.isLP64` wasn't set here in `frontend.d` at all, remaining at its `false` initial value.